xchange-coinbasepro/src/main/java/org/knowm/xchange/coinbasepro/CoinbaseProExchange.java

```diff
@@ -16,6 +16,7 @@
 import lombok.NoArgsConstructor;
```

Comment:
why did the entirety of this class get highlighted as changes?

Reply:
Ah. I think I edited this file before I added the google code format plugin. I will fix this up as expected.